jobs/signing/sign-artifacts/Jenkinsfile

```diff
@@ -363,7 +377,13 @@ node {
                                 sh "aws s3 sync --no-progress --exclude='*' --include 'sha256=*' ./ s3://art-srv-enterprise/pub/rhacs/signatures/rh-acs/"
                             }
                         }
-
+                    } else if ( params.PRODUCT == 'coreos-installer' ) {
+                        sshagent(["openshift-bot"]) {
```

Comment:
sshagent is not necessary.

Reply:
I saw it used in other product, I don't know if we need to prune them all